test/plots/areaPlotTests.ts

```diff
@@ -2,162 +2,181 @@
+        areaPlot.datasets().forEach((dataset: Plottable.Dataset) => {
+          let data = dataset.data();
+          let pointsInArea = getPointsInArea(areaPlot, data);
+          TestMethods.assertLinePathEqualToDataPoints(areaPath.attr("d"), pointsInArea, xScale, yScale);
+          assert.strictEqual(areaPath.attr("fill"), fill, "area fill was set correctly");
+          assert.strictEqual(areaPath.style("stroke"), "none", "area stroke renders as \"none\"");
+
+          let linePath = content.select(".line");
+          TestMethods.assertLinePathEqualToDataPoints(linePath.attr("d"), data, xScale, yScale);
+          assert.strictEqual(linePath.attr("stroke"), "#000000", "line stroke was set correctly");
+          assert.strictEqual(linePath.style("fill"), "none", "line fill renders as \"none\"");
+        });
+        svg.remove();
+      });
 
-    it("area fill works for non-zero floor values appropriately, e.g. half the height of the line", () => {
-      areaPlot.y0((d) => d.bar / 2);
-      areaPlot.renderTo(svg);
-      renderArea = (<any> areaPlot)._renderArea;
-      let areaPath = renderArea.select(".area");
-      assert.strictEqual(TestMethods.normalizePath(areaPath.attr("d")), "M0,500L500,0L500,250L0,500Z");
-      svg.remove();
-    });
+      it("fills for non-zero floor values appropriately", () => {
+        areaPlot.y0((d) => d.y / 2);
+        areaPlot.renderTo(svg);
+        areaPlot.datasets().forEach((dataset: Plottable.Dataset) => {
+          let data = dataset.data();
+          let areaPath = areaPlot.content().select(".area");
+          let pointsInArea = getPointsInArea(areaPlot, data);
+          TestMethods.assertLinePathEqualToDataPoints(areaPath.attr("d"), pointsInArea, xScale, yScale);
+        });
+        svg.remove();
+      });
 
-    it("area is appended before line", () => {
-      let paths = renderArea.selectAll("path")[0];
-      let areaSelection = renderArea.select(".area")[0][0];
-      let lineSelection = renderArea.select(".line")[0][0];
-      assert.operator(paths.indexOf(areaSelection), "<", paths.indexOf(lineSelection), "area appended before line");
-      svg.remove();
-    });
+      it("places the area before line", () => {
+        let content = areaPlot.content();
+        let paths = content.selectAll("path")[0];
+        let areaElement = content.select(".area")[0][0];
+        let lineElement = content.select(".line")[0][0];
+        assert.operator(paths.indexOf(areaElement), "<", paths.indexOf(lineElement), "area appended before line");
+        svg.remove();
+      });
 
-    it("cleans up correctly when removing Datasets", () => {
-      areaPlot.renderTo(svg);
-      areaPlot.removeDataset(simpleDataset);
-      let paths = areaPlot.content().selectAll("path");
-      assert.strictEqual(paths.size(), 0, "removing a Dataset cleans up all <path>s associated with it");
-      svg.remove();
-    });
+      it("cleans up correctly when removing Datasets", () => {
```

Comment:
Note that I am growing to be more against using the word "correctly" since it is a really vague term.  Let's define what "correctly" means and then use that as the test name